e2e/scenarios/kops-simple

```diff
@@ -125,18 +135,17 @@ fi
 
 kubetest2 ${KUBETEST2_ARGS} \
   --up \
-   \
+  --build \
```

Comment:
(--build will build kOps, which is needed because otherwise we can be mixing versions here, between the binary we built above and supporting binaries like the on-node agent)